ambari-server/src/main/java/org/apache/ambari/server/controller/internal/Stack.java

```diff
@@ -422,11 +414,6 @@ static String formatMissingServiceForConfigType(String config, String stackId) {
         Collections.emptySet();
   }
 
-  @Override
```

Comment:
`dependencyConditionalServiceMap` was a "read-only" map: always empty, because production code only queried it, never put any items into it.  I removed it in a separate commit d5347f6ae2, which can be inspected to verify this claim.

Reply:
ok, thanks for clarifying this.